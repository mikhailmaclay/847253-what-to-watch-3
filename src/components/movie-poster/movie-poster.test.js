// Libraries
import React from 'react';
import renderer from 'react-test-renderer';
// Components
import MoviePoster from './movie-poster';

const src = `src`;
const alt = `alt`;

it(`<MoviePoster/> should render correctly`, () => {
  const result = renderer.create(
      <MoviePoster src={src} alt={alt}/>
  ).toJSON();

  expect(result).toMatchSnapshot();
});
