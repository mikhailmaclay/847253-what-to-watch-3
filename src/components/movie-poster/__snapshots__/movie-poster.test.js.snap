// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoviePoster/> should render correctly 1`] = `
<div>
  <img
    alt="alt"
    height="327"
    src="src"
    width="218"
  />
</div>
`;
